tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "noLib": false,
     "outDir": "./dist",
     "removeComments": true,
+    "strict": true,
+    "strictPropertyInitialization": false,
```

Comment:
@rkirov does it work for you if we disable `strictPropertyInitialization`?

Reply:
@mgechev There are only few some places that forced me to disable this. I can enable this and initialize properties properly or even use a `!` to indicate that I want _hack_ that property. Let me know what should I do.

Reply:
If the job isn't too hard, I would prefer use the default value of `strictPropertyInitialization`. Could you update this PR in this direction ?

Reply:
@wKoza Updated.